app/models/metric/statistic.rb

```diff
@@ -0,0 +1,32 @@
+module Metric::Statistic
+  STAT_COLS = [
+    :stat_containergroup_create_rate,
+  ]
+
+  VALID_STAT_TARGETS = [
+    ExtManagementSystem,
+  ]
+
+  def self.calculate_stat_columns(obj, attrs, ts = nil)
+    valid_obj = VALID_STAT_TARGETS.any? { |t| obj.kind_of?(t) }
+    raise "object #{obj} is not one of #{VALID_STAT_TARGETS.collect(&:name).join(", ")}" unless valid_obj
+
+    ts = attrs[:timestamp] if ts.nil?
+    state = obj.vim_performance_state_for_ts(ts)
+
+    end_time = Time.parse(ts).utc
+    start_time = end_time - state[:capture_interval]
+
+    result = {}
+
+    STAT_COLS.each do |col|
+      _, _group, typ, _mode = col.to_s.split("_")
+      case typ
+      when "create"
+        result[col] = obj.container_groups.where(:ems_created_on => start_time..end_time).count
```

Comment:
This line will query only container groups that werent deleted. 
If you need to get the deleted ones as well, you could run ```obj.container_groups.unscope.where(...)```